src/modules/io/om.py

```diff
@@ -10,8 +10,8 @@ def __init__(self):
     def importGeometry(self, fileName):
         g = Geometry()
         try:
-            m = om.read_trimesh(fileName)
-            # m = om.read_polymesh(fileName)
+            # m = om.read_trimesh(fileName)
+            m = om.read_polymesh(fileName)
```

Comment:
It would  be ideal if this code snippet would read desired behaviour from the configuration, and act acordingly....

If you agree, I'll do the necessary changes and push it to your fork?